typed-protocols/src/Network/TypedProtocol/PingPong/Tests.hs

```diff
@@ -269,10 +271,13 @@ prop_connect_pipelined5 choices (Positive omax) (NonNegative n) =
 -- | Run a non-pipelined client and server over a channel using a codec.
 --
 prop_channel :: (MonadSTM m, MonadAsync m, MonadCatch m)
-             => NonNegative Int -> m Bool
-prop_channel (NonNegative n) = do
+             => NonNegative Int -> Tracer m String -> m Bool
+prop_channel (NonNegative n) tr = do
+    tracingWith tr "Starting ..."
     ((), n') <- runConnectedPeers createConnectedChannels
+                                  tr
                                   codecPingPong client server
+    tracingWith tr "... ending."
```

Comment:
What's the usefulness of this and the `"Starting ..." trace?

Reply:
Just to indicate the start and the end of this process. Removed them for cleaner code.